modules/nf-core/ltrfinder/main.nf

```diff
@@ -0,0 +1,53 @@
+process LTRFINDER {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/edta:2.2.0--hdfd78af_0':
+        'biocontainers/edta:2.2.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("*.scn")      , emit: scn
+    tuple val(meta), path("*.gff3")     , emit: gff
+    path "versions.yml"                 , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    /usr/local/share/EDTA/bin/LTR_FINDER_parallel/LTR_FINDER_parallel \\
```

Comment:
Does this work with the conda env? That will break if installed anywhere else. Compare with the Busco module if the path needs to sourced from somewhere in the module.

Reply:
Thanks for highlighting this. I don't test with conda locally as it is excruciatingly slow. The CI workflow did not catch it because it has not been executing CONDA test since last week for some of the pull requests.

Now I have used a relative path and tested that path with conda locally.